contracts/UniswapV3Staker.sol

```diff
@@ -253,56 +235,74 @@ contract UniswapV3Staker is
                 params.claimDeadline
             );
 
+        require(
```

Comment:
TODO: We should make sure this is covered by a unit test

Reply:
I'll work on getting the unit tests done for these